.github/workflows/node.js.yml

```diff
@@ -25,4 +25,7 @@ jobs:
       with:
         node-version: 14.x
     - run: yarn
-    - run: yarn test
+    - run: cd packages/web-components && yarn
+    - run: cd packages/web-components && yarn build
+    - run: cd packages/react-components && yarn
+    - run: cd packages/react-components && yarn test
```

Comment:
This is part of a required check, so I wanted to make sure it passes. This will:

1. Install dependencies & build `web-components` (it is used in `react-components`)
2. Install dependencies for `react-components`
3. Run the tests for `react-components`